react/features/etherpad/components/native/styles.js

```diff
@@ -1,22 +1,29 @@
 // @flow
 
-import { ColorPalette } from '../../../base/styles';
+import BaseTheme from '../../../base/ui/components/BaseTheme.native';
 
-export const INDICATOR_COLOR = ColorPalette.lightGrey;
+
+export const INDICATOR_COLOR = BaseTheme.palette.indicatorColor;
 
 export default {
     indicatorWrapper: {
         alignItems: 'center',
-        backgroundColor: ColorPalette.white,
+        backgroundColor: BaseTheme.palette.ui12,
         height: '100%',
         justifyContent: 'center'
     },
 
     sharedDocContainer: {
-        flex: 1
+        backgroundColor: BaseTheme.palette.ui12,
+        flex: 1,
+        paddingRight: BaseTheme.spacing[3]
+    },
+
+    sharedDoc: {
+        marginBottom: BaseTheme.spacing[3]
```

Comment:
What about devices without a notch? We don't want it there do we?

Reply:
![image](https://user-images.githubusercontent.com/26413496/177332321-937d113a-6e6a-4e60-9b5c-c658554ef2b1.png)

![image](https://user-images.githubusercontent.com/26413496/177332260-b1879c88-e2b2-4e12-bfce-9b5af73ddc94.png)